src/example/Simplest.js

```diff
@@ -0,0 +1,71 @@
+import ActionManager from "../action/ActionManager.js";
+import DoubleJumpFilter from "./filters/DoubleJumpFilter.js";
+
+function main() {
+  // App specific setup of the ActionManager
+
+  let actionManager = new ActionManager();
+
+  // Load a custom filter
+  actionManager.loadFilter(DoubleJumpFilter, "/filter/double-jump");
+
+  // Load up the maps that bind inputs to filters and actions
+  actionManager
+    .loadActionMaps(
+      // Maps used while the mode is flat
+      "./flat-playing.map",
```

Comment:
What about explicitely link the actions to modes:
```
.loadActionMaps('flat', ...)
.loadActionMaps('portal', ...)
.loadActionMaps('immersive', ...)
```
so with that the app could just switch between `paused`, `menu` and `playing` and the `actionManager` will choose the required action based on the current app mode.

Reply:
Yes, I was thinking about this. I'm torn about whether or not to embed the "flat", "portal", and "immersive" display mode concepts into the library. On the one hand, I think it could be great to have standard modes that devs comprehend once and that eases sharing and extending action maps. On the other hand, what if people have very different modes that they want to support?
What do you think?
Either way, I like the idea of having maps grouped by mode (and perhaps scenario like playing, paused, etc) to make it easy for the code to toggle them in and out.

Reply:
I won't include the `flat`, `portal` or `immersive` mode explicitly on the lib, maybe just on the examples, as not everyone using the library will be targeting XR experiences.
But I believe adding a `mode` that you could define what does it means has a nice value. So you could have something like `actionManager.registerModes('flat', 'portal', 'immersive');` and then it's your responsibility when coding the app to switch between these modes depending on the context.

Reply:
I like this direction. Since we'll be including and maintaining action maps for common modes and scenarios, we'll have to make some decisions about those, but the library itself can be written to work with different mode and scenario choices.